src/Components/Spotify/SpotifyAuth.js

```diff
@@ -0,0 +1,13 @@
+import spotifyConfig from "../../Config/spotify";
+
+
+const SpotifyAuth = function () {
+  return (
+    <a className={""}
```

Comment:
Why the empty className?

Reply:
I already put it there for future css styling. I'll remove it and we can add it later